app/discord/bot.ts

```
@@ -0,0 +1,178 @@
+                }
+                return;
+            }
+        }
+
+        let userText: string | undefined = message.content;
+        let imageDataInput: any;
+
+        // Handle attachments (images)
+        if (message.attachments.size > 0) {
+            const attachment = message.attachments.first();
+            if (attachment && attachment.url) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)